packages/jellyfish-api-core/src/category/blockchain.ts

```diff
@@ -306,3 +346,42 @@ export interface MempoolTx {
     'bip125-replaceable': boolean
   }
 }
+
+
+export interface BlockStats {
+  avgfee: number
+  avgfeerate: number
+  avgtxsize: number
+  blockhash: number
+  height: number
+  ins: number
+  maxfee: number
+  maxfeerate: number
+  maxtxsize: number
+  medianfee: number
+  mediantime: number
+  mediantxsize: number
+  minfee: number
+  minfeerate: number
+  mintxsize: number
+  outs: number
+  subsidy: number
+  swtxs: number
+  time: number
+  totalfee: number
+  txs: number
+  "swtotal_size": number
+  "swtotal_weight": number
+  "total_out": number
+  "total_size": number
+  "total_weight": number
+  "utxo_increase": number
+  "utxo_size_inc": number
+  "feerate_percentiles": {
+    '10th_percentile_feerate': number
+    '25th_percentile_feerate': number
+    '50th_percentile_feerate': number
+    '75th_percentile_feerate': number
+    '90th_percentile_feerate': number
+  } 
```

Comment:
Remove `""` and `''`, don't need it.

Reply:
Oh my bad. Removed the `""`, but for the `feerate_percentiles`, we need `''`

Reply:
> Oh my bad. Removed the `""`, but for the `feerate_percentiles`, we need `''`

Oh yeah, misread, didn't see it start with a number